backend/init_data/skills/browser/screenshot_tool.py

```
@@ -0,0 +1,276 @@
+            if type == "jpeg" and quality is not None:
+                screenshot_options["quality"] = max(0, min(100, quality))
+
+            # Take screenshot
+            if selector:
+                # Screenshot specific element
+                element = page.locator(selector)
+                await element.wait_for(timeout=self.default_timeout)
+                screenshot_bytes = await element.screenshot(**screenshot_options)
+            else:
+                # Screenshot page
+                screenshot_bytes = await page.screenshot(**screenshot_options)
```

Ruff (0.14.14)

[error] Local variable `screenshot_bytes` is assigned to but never used

Remove assignment to unused variable `screenshot_bytes`

(F841)